README.md

```diff
@@ -200,12 +219,81 @@ mstorage.info("<media_id>"){result, error in
 }
 ```
 
+### Attach media metadata
+You can define your original metadata as a 'user metadata'.
+Existing metadata value for the same key will be overwritten. Up to 10 user metadata can be attached to a media data. 
+
+```swift
+mstorage.addMeta("<media_id>", ["user.<key1>": "<value1>", "user.<key2>": "<value2>"]){error in
+    if error.isEmpty() {
+        // do something
+    }
+}
+```
+
 ### Get media metadata
+* All
 ```swift
 mstorage.meta("<media_id>"){result, error in
     if error.isEmpty() {
         var exif: [String: String] = result.exif
         var gpano: [String: String] = result.gpano
+        var userMeta: [String: String] = result.userMeta
+        // do something
+    }
+}
+```
+
+* Exif
+```swift
+mstorage.meta("<media_id>", "exif"){result, error in
+    if error.isEmpty() {
+        // do something
```

Comment:
I want a line like `var exif: [String: String] = result.exif` to know how the `result` has the exif data. (It can be ` = result.exif` OR  `= result`)